pants.ini

```diff
@@ -322,6 +322,16 @@ branch_notes: {
     '1.6.x': 'src/python/pants/notes/1.6.x.rst',
   }
 
+[shader]
+binary_package_excludes: [
```

Comment:
It would be good to put all of these in the default list, and then folks can add and remove if they'd like?

If you need to adjust them, we have the set syntax, so you can do subtractions and additions. For example:
```
[shader]
binary_package_excludes: -[
    'com.oracle',
  ]
```